src/patternfly/components/Chips/chip.scss

```diff
@@ -0,0 +1,185 @@
+
+  position: relative;
+  display: inline-flex;
+  padding-left: var(--pf-c-chip--PaddingLeft);
+  color: var(--pf-c-chip--Color);
+  vertical-align: middle;
+  background-color: var(--pf-c-chip--BackgroundColor);
+  border: 0;
+  border-radius: var(--pf-c-chip--BorderRadius);
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border: var(--pf-c-chip--BorderWidth) solid var(--pf-c-chip--BorderColor);
+    border-radius: var(--pf-c-chip--BorderRadius);
+  }
+
+  &:not(:last-child) {
+    margin-right: var(--pf-c-chip--MarginRight);
+  }
+
+  // Chip Hover State
+  &.pf-m-hover,
+  &:hover {
+    .pf-c-button {
+      color: var(--pf-c-chip__button--hover--Color);
+    }
+
+    &::before {
+      border-color: var(--pf-c-chip--hover--BorderColor);
+    }
+  }
+
+  // Chip Overflow Modifier
+  &.pf-m-overflow {
+    padding-left: 0;
+    background-color: var(--pf-c-chip--m-overflow--BackgroundColor);
+
+    // Remove Chip border
+    &::before {
+      border: 0 transparent;
+    }
+
+    .pf-c-button {
+      padding-right: var(--pf-c-chip--m-overflow__button--PaddingRight);
+      padding-left: var(--pf-c-chip--m-overflow__button--PaddingLeft);
+
+      &.pf-m-active,
+      &:active::after {
+        border: var(--pf-c-chip__button--active--BorderWidth) solid var(--pf-c-chip__button--active--BorderColor);
+        border-radius: var(--pf-c-chip__button--BorderRadius);
+      }
+
+      &.pf-m-focus,
+      &:focus::after {
+        border: var(--pf-c-chip__button--focus--BorderWidth) solid var(--pf-c-chip__button--focus--BorderColor);
+        border-radius: var(--pf-c-chip__button--BorderRadius);
+      }
+    }
+
+    &.pf-m-hover,
+    &:hover {
+      &::before {
+        border: var(--pf-c-chip__button--hover--BorderWidth) solid var(--pf-c-chip__button--hover--BorderColor);
+      }
+    }
+  }
+
+  // Text Element
+  .pf-c-chip__text {
+    max-width: var(--pf-c-chip__text--MaxWidth);
+    margin: auto;
+    overflow: hidden;
+    font-size: var(--pf-c-chip__text--FontSize);
+    color: var(--pf-c-chip__text--Color);
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+
+  // Remove Button
+  .pf-c-button {
+    min-width: var(--pf-c-chip__button--MinWidth);
+    min-height: var(--pf-c-chip__button--MinHeight);
+    padding-right: var(--pf-c-chip__button--PaddingRight);
+    padding-left: var(--pf-c-chip__button--PaddingLeft);
+    font-size: var(--pf-c-chip__button--FontSize);
+    color: var(--pf-c-chip__button--Color);
+
+    &.pf-m-focus,
+    &:focus {
+      color: var(--pf-c-chip__button--focus--Color);
+      outline: none; // turn off default button focus styles
+    }
+
+    &.pf-m-active,
+    &:active {
+      color: var(--pf-c-chip__button--active--Color);
+    }
+
+    // Turn off default button margin-right style
+    > :first-child {
+      margin-right: 0;
+    }
+  }
+}
+
+// Chip group container
+.pf-c-chip__group {
```

Comment:
Hmm... it's kind of odd to have `.pf-c-chip__group` as a parent of `.pf-c-chip`. by the CSS, the block is "chip" and the element is "group", but by the markup, the block is the chip group and the element is the chip.

I wonder if a chip group and a chip should be separate things, like the input group and inputs.

Reply:
@mcoker You're definitely right I wasn't sure what to do here. @matthewcarleton what do you think? Im happy to make another component that groups the chips.

Reply:
I concur. Because of the styles introduced I think the chip group should be a new component.